gwtmockito/src/main/java/com/google/gwtmockito/GwtMockitoTestRunner.java

```diff
@@ -263,6 +263,7 @@ public GwtMockitoTestRunner(Class<?> unitTestClass) throws InitializationError {
     packages.add("org.jacoco"); // To support JaCoCo code coverage tools
     packages.add("org.hamcrest"); // Since this package is referenced directly from org.junit
     packages.add("org.junit"); // Make sure the ParentRunner can recognize annotations like @Test
+    packages.add("jdk.internal.reflect"); // Java9 loading mechanism
```

Comment:
Would you mind moving this up between lines 261 and 262 so that the list stays alphabetical?

Reply:
OK I will do it

Reply:
Hello @ekuefler , I've done the change